### ISSUE TITLE
Review for Common Data Objects

### ISSUE THREAD

**Issue by ePaul** [zalando/restful-api-guidelines#51]
*(This is a part of #41, see there for an overview.)*

Commit a68803b1 [changed these things in the Common Data Objects section](https://github.com/zalando/restful-api-guidelines/commit/a68803b1ed6025d37200e8d456075488f7d51068?short_path=f438b86#diff-f438b863a8a28c230aca1e0eff90af71):

* common address object:
   * wording change in introductory paragraph
   * extended description for salutation
   * change `#` to `no`
* common error return object: add "Hint: APIs may define custom problem types with extension properties, according to their specific needs.".
* change reasoning for the "no stack trace" rule.

Please review the diff and comment here.

**Comment by tkrop** [comment]
Reviewed it and added comments to commit. Please review comments.

**Comment by ePaul** [comment]
* I'll fix the "apt no" in a bundled pull request with editorial changes.
* The duplicated sentence in the problem/error object was already fixed in #3.

This
 > * Other teams would possibly rely on the stack trace and a refactoring would be hard.
 > * Brand partners and other third parties receiving the API, might receive leaked, sensitive
 > information

was replaced by this:

> * Clients may not rely on pure API implementation details. 

@tkrop commented:

> Why is this line deleted? Do we not care about leaking sensitive information, or is there a general  filter in place stripping of this information?
> I think it is still true: Stack traces may leak sensitive information that can be use by attackers.

I'm not aware of any such filter, and I guess I agree. How can we reword it to make it better?

**Comment by ePaul** [comment]
With #53 and #64 being merged, this can be closed.
